src/styles.scss

```diff
@@ -1,21 +1,42 @@
-@import './app/shared/variables/global.scss';
+@import "~@angular/material/prebuilt-themes/indigo-pink.css";
+@import "./app/shared/variables/global.scss";
+
+@font-face {
+  font-family: Open Sans;
+  src: url(assets/fonts/OpenSans.ttf) format("truetype");
+}
 
 html {
   margin: 0;
   padding: 0;
   height: 100%;
+  background-color: $color-secondary;
 }
 
 body {
   background: linear-gradient(
     178.38deg,
-    #e74850 13.52%,
-    #236fe9 34.61%,
-    #4286f4 69.76%
+    $color-primary 13.52%,
+    $color-secondary-ligth 34.61%,
+    $color-secondary 69.76%
   );
   margin: 0;
+  padding: 0;
   font-size: 14px;
-  font-family: "Open Sans", sans-serif;
+  font-family: $font-family;
   font-weight: 300;
-  color: #000;
-}
\ No newline at end of file
+  color: $color-text-dark;
+  overflow: -moz-scrollbars-vertical; 
+  overflow-y: scroll;
+}
+
+a {
+  text-decoration: none;
+}
+
+.no-padding {
+  padding: 0;
+}
+
+html, body { height: 100%; }
```

Comment:
html and body you have in lines 9 i 16

Reply:
cd4bac36abdf4c7132b5785c4804c58ea3ada3ac